### PR TITLE
Canu recipe.

### PR DESCRIPTION
**Pull request by jxtx**
Builds and tests, but probably missing runtime dependencies in Perl modules
I have not yet hit.

### CODE REVIEW

File: recipes/canu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  # Canu currently needs to come from git to get it's version string right
+  git_url: https://github.com/marbl/canu.git
+  git_rev: v1.1
+
+build:
+    detect_binary_files_with_prefix: True
+
+requirements:
+    run:
+        - perl-filesys-df
```
Comment (bgruening) [inline recipes/canu/meta.yaml]: `- perl-threaded     # [not osx]`

File: recipes/canu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package:
+  name: canu
+  version: 1.1
+
+source:
+  # Canu currently needs to come from git to get it's version string right
+  git_url: https://github.com/marbl/canu.git
```
Comment (bgruening) [inline recipes/canu/meta.yaml]: Can we use a downloadable url here?

File: recipes/canu/meta.yaml (hunk 1 above)
Comment (jxtx) [inline recipes/canu/meta.yaml]: Fixed.

File: recipes/canu/meta.yaml (hunk 2 above)
Comment (jxtx) [inline recipes/canu/meta.yaml]: If not running in a git checkout Canu does not set it's version information. We could hack out the canu_version_update.pl step and manually create canu_version.H if that's preferable (seemed ugly to me).

**Comment by johanneskoester** [comment]
Sorry for the delay, can you update the branch with master?

**Comment by bgruening** [comment]
@jxtx this is deprecated isn't it?

**Comment by jxtx** [comment]
Since Bjoern already merged this recipe with his perl changes, this is
redundant and can be closed.

**Pull request closed without merging**